streamly-bytestring.cabal

```diff
@@ -28,7 +28,7 @@ library
       Paths_streamly_bytestring
   hs-source-dirs:
       src
-  ghc-options: -Wall -O2
+  ghc-options: -Werror -Wall -O2
```

Comment:
This is not good at all. Should not add -Werror for all the users of the library. We should use a separate cabal.project file to do it only in CI.

Reply:
This was fixed in another PR.